docs/changelogs/CHANGELOG-2.26.md

```diff
@@ -0,0 +1,320 @@
+- Add Baremetal provider and Tinkerbell Support ([#6764](https://github.com/kubermatic/dashboard/pull/6764))
+- Add the grafana orgId parameter to Grafana UI link in dashboard ([#6617](https://github.com/kubermatic/dashboard/pull/6617))
+- Audit logging backend webhook configuration for cluster and datacenter ([#6781](https://github.com/kubermatic/dashboard/pull/6781))
+- Cluster Backup: CA bundle and Prefix configuration for backup storage ([#6682](https://github.com/kubermatic/dashboard/pull/6682))
+- Display the used preset name on the cluster detail page ([#6705](https://github.com/kubermatic/dashboard/pull/6705))
+- Enable editing allowed IP ranges for NodePorts ([#6783](https://github.com/kubermatic/dashboard/pull/6783))
+- Support for configuring internet access for the web terminal ([#6668](https://github.com/kubermatic/dashboard/pull/6668))
+- Support for enabling/disabling operating systems for machines in user clusters at the project level ([#6723](https://github.com/kubermatic/dashboard/pull/6723)) 
+- Update KubeLB integration to support enabling/disabling gateway API and load balancer class ([#6810](https://github.com/kubermatic/dashboard/pull/6810))
+- Update to Angular version 17 ([#6639](https://github.com/kubermatic/dashboard/pull/6639))
+- Admin panel settings for applications ([#6787](https://github.com/kubermatic/dashboard/pull/6787))
+    - Admins can now manage applications using admin panel
+    - Application can be marked as default or enforced using dashboard
+
+#### Bugfixes
+
+- Adjust the preset domain field to accept emails ([#6690](https://github.com/kubermatic/dashboard/pull/6690))
+- Fix a bug where CNI was always being defaulted to cilium irrespective of what was configured in the cluster template or default cluster template ([#6708](https://github.com/kubermatic/dashboard/pull/6708))
+- Fix an issue where the cursor in web terminal kept jumping to the beginning due to sizing issue ([#6799](https://github.com/kubermatic/dashboard/pull/6799))     
+- Fix template value for machine deployments in edit mode ([#6669](https://github.com/kubermatic/dashboard/pull/6669))
+- Fix the pagination in project members table ([#6741](https://github.com/kubermatic/dashboard/pull/6741))
+- Fix TLS errors in the admin page when using a custom CA for the metering object store ([#6752](https://github.com/kubermatic/dashboard/pull/6752))
+- Grant admin all owner Privileges on all projects ([#6754](https://github.com/kubermatic/dashboard/pull/6754))
+- Resolve conflict in determining available Kubernetes versions where upgrades where possible in `Cluster` object but not via the Dashboard ([#6651](https://github.com/kubermatic/dashboard/pull/6651))
+- Support for eBPF proxy mode when the CNI plugin is none ([#6757](https://github.com/kubermatic/dashboard/pull/6757))
+
+#### Updates
+
+- Update Go version to 1.22.5 ([#6729](https://github.com/kubermatic/dashboard/pull/6729))
```

Comment:
This should be replaced when the changelog is updated; the dashboard was updated to Go 1.23.1 today.